### PR TITLE
Circular Progress Bard Component

### PR DESCRIPTION
**Pull request by eldiegod**
#327

### CODE REVIEW

File: src/shared/components/CircularProgressbar/CircularProgressbar.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+export const VIEWBOX_WIDTH = 100
+export const VIEWBOX_HEIGHT = 100
+export const VIEWBOX_HEIGHT_HALF = 50
+export const VIEWBOX_CENTER_X = 50
+export const VIEWBOX_CENTER_Y = 50
+
+export type CircularProgressbarProps = {
+  value: number
+  circleRatio: number
```
Comment (kdembler) [inline src/shared/components/CircularProgressbar/CircularProgressbar.tsx:12]: What does this represent?

File: src/shared/components/CircularProgressbar/CircularProgressbar.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import * as React from 'react'
+import { SVG, Background, Trail, StyledPath } from './CircularProgressbar.style'
+
+export const VIEWBOX_WIDTH = 100
+export const VIEWBOX_HEIGHT = 100
+export const VIEWBOX_HEIGHT_HALF = 50
```
Comment (kdembler) [inline src/shared/components/CircularProgressbar/CircularProgressbar.tsx]:
```suggestion
export const VIEWBOX_HEIGHT_HALF = VIEWBOX_HEIGHT / 2
```

File: src/shared/components/CircularProgressbar/CircularProgressbar.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+export const VIEWBOX_CENTER_X = 50
```
Comment (kdembler) [inline src/shared/components/CircularProgressbar/CircularProgressbar.tsx]: Aren't those `height(width) / 2` as well?

Comment (eldiegod) [reply src/shared/components/CircularProgressbar/CircularProgressbar.tsx]: Yup it is

File: src/shared/components/CircularProgressbar/CircularProgressbar.tsx (hunk 1 above)
Comment (eldiegod) [reply src/shared/components/CircularProgressbar/CircularProgressbar.tsx:12]: Is how much of the circle the progress bar will cover so 1 for the whole circle or maybe 0.5 for half the circle so at the max value the progressbar will only cover half the circle

**Review by kdembler — APPROVED** [review]
Looks real nice!

**Pull request merged**